SparkPost/2web2ui
# AC-1504: (UI)Flatted Onboarding Design

AC-1504: (UI)Flatted Onboarding Design

### What Changed
 - Removed breadcrumbs in the Getting Started Guide.
 - Flattened the Step to be in Expandables as in mocks https://sparkpost.invisionapp.com/public/share/E415G849MB#/screens/412111027
- Check if the "Start Sending with SparkPost" is open By Default.
- **Note** - Part of Getting started guide is now visible to template and reporting users as well
![Screen Shot 2020-06-16 at 3 22 08 PM](https://user-images.githubusercontent.com/4179337/84833244-dcd83e00-affc-11ea-921d-6427df4f9aeb.png)

### How To Test
 - Check if the implementation matches the mocks
 - Try different roles - template, developer, reporting etc

### To Do
